app/src/main/java/org/simple/clinic/di/network/NetworkModule.kt

```diff
@@ -19,18 +19,18 @@ import org.simple.clinic.patient.businessid.BusinessId
 import org.simple.clinic.patient.businessid.Identifier
```

Comment:
Just an FYI, we should not include a random format to a file that is not part of the PR.